src/pages/Home/Home.tsx

```diff
@@ -1,10 +1,12 @@
 import { Hero } from '@/components/Hero'
+import { Technologies } from '@/components/Technologies/Technologies'
```

Comment:
You can remove the las `/Technologies'` of the import

Reply:
True